ansible/configs/kni-osp/post_software.yml

```diff
@@ -0,0 +1,57 @@
+- name: Step 005 - Show me what you got !
+  when: my_auto_install
+  hosts: bastions
+  become: false
+  gather_facts: false
+  environment:
+    KUBECONFIG: /home/{{ ansible_user }}/scripts/ocp/auth/kubeconfig
+  tasks:
+  - debug:
+      msg: "Post-Software Steps starting"
+
+  - name: Get kubeadmin password
+    slurp:
+      path: /home/{{ ansible_user }}/scripts/ocp/auth/kubeadmin-password
+    register: kubeadminr
+
+  - name: Get console route
+    environment:
+      KUBECONFIG: /home/{{ ansible_user }}/scripts/ocp/auth/kubeconfig
+    command: oc get route -n openshift-console console -o json
+    register: routeconsole
+    retries: 10
+    delay: 30
+    until: routeconsole is succeeded
+    ignore_errors: yes
+
+  - name: Get Webconsole URL
```

Comment:
Do you need this if you are already getting the route for the console in the previous task?